backend/src/main/java/com/tradomator/model/Balance.java

```diff
@@ -0,0 +1,18 @@
+package com.tradomator.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Builder
+public class Balance {
+
+    private String asset;
+    private String free;
```

Comment:
free and locked sound like two opposite attributes. Do you really need both of them? And shouldn't they rather be booleans?

Reply:
that's true. thanks. For now i just keep them unchanged as i revise them directly from external api. For integrating further logic it makes sense to adjust to boolean. Cheers T